core/src/test/java/org/bitcoinj/core/LockTimeTest.java

```diff
@@ -73,11 +76,32 @@ public void unset() {
 
     @Test(expected = IllegalStateException.class)
     public void blockHeight_mismatch() {
-        LockTime.ofTimestamp(Instant.MAX).blockHeight();
+        LockTime.ofTimestamp(Instant.MAX).getBlockHeight().orElseThrow(IllegalStateException::new);
```

Comment:
`.orElseThrow()`  requires Java 9 API I think. Maybe we should make a clear statement about the tests in core. I wouldn't mind switching both language features and API officially over to Java 11.

Reply:
Java 9 provides `.orElseThrow()` with no arguments. The above is valid Java 8.